docs/VERSIONS.md

```diff
@@ -1 +1,47 @@
 # Historical Versions
+
+## tag 0.2.1 (testnet)
+
+**spore**:
+| | |
+|-|-|
+|tx_hash|0x06995b9fc19461a2bf9933e57b69af47a20bf0a5bc6c0ffcb85567a2c733f0a1|
+|index|0|
+|code_hash|0x5e063b4c0e7abeaa6a428df3b693521a3050934cf3b0ae97a800d1bc31449398|
+
+**cluster**:
+| | |
+|-|-|
+|tx_hash|0xfbceb70b2e683ef3a97865bb88e082e3e5366ee195a9c826e3c07d1026792fcd|
+|index|0|
+|code_hash|0x7366a61534fa7c7e6225ecc0d828ea3b5366adec2b58206f2ee84995fe030075|
+
+**cluster_agent**:
+| | |
+|-|-|
+|tx_hash|0x53fdb9366637434ff685d0aca5e2a68a859b6fcaa4b608a7ecca0713fed0f5b7|
+|index|0|
+|code_hash|0xc986099b41d79ca1b2a56ce5874bcda8175440a17298ea5e2bbc3897736b8c21|
+
+**cluster_proxy**:
+| | |
+|-|-|
+|tx_hash|0x0231ea581bbc38965e10a2659da326ae840c038a9d0d6849f458b51d94870104|
+|index|0|
+|code_hash|0xbe8b9ce3d05a32c4bb26fe71cd5fc1407ce91e3a8b9e8719be2ab072cef1454b|
+
+## tag 0.2.2 (mainnet)
+
+**spore**:
+| | |
+|-|-|
+|tx_hash|0xcb67e11a39594a183eae50664073a85fa0140ae25480df61114aca0e47726d38|
+|index|0|
+|code_hash|0x9cb02fa8621d79fff300b910d52665f594d3ad468142a66213cfaaca0409502c|
```

Comment:
maybe this code_hash should be deleted?
@ashuralyk

Reply:
**suggestions for next steps:**
1. spore-contract/pull/52
2. build and test
3. create a tag on master branch  for mainnet
4. check the codehash, do the redeployment on mainnet
5. create a new PR to update the deployment info, such as `VERSIONs.md`